common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -132,13 +150,13 @@
             "error"
         ],
         "@typescript-eslint/no-empty-function": [
-            "off"
+            "error"
```

Comment:
I don't think I like no-empty-function at first glance -- maybe just warn?

Reply:
What's your philosophy on warnings? How do you decide between warning and error?

FWIW the issues that this has flagged seem somewhat suspicious. Why have a function that does nothing?

Reply:
Made a warning.

Reply:
Mocks, abstract methods, and empty constructors when using parameter properties seem reasonable.

I'm not super opposed to it since a comment is sufficient to clear the error, but seems kind of annoying for the more obvious cases.